media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -42,9 +44,10 @@ BDEPEND="
 "
 RDEPEND="
 	media-libs/alsa-lib
-	sys-apps/dbus
+	sys-apps/dbus[${MULTILIB_USEDEP}]
 	sys-libs/ncurses[unicode]
```

Comment:
No multilib needed for ncurses?

Reply:
Currently ncurses is only used for pw-top tool, which will be overwritten if built for non-native ABIs anyway. And if multilib for it was missing, it should merely silently disable building of pw-top from what I recall about the build system:
$ grep ncurses /var/db/pkg/media-video/pipewire-9999/NEEDED
/usr/bin/pw-top libpipewire-0.3.so.0,libncursesw.so.6,libtinfow.so.6,libc.so.6